Jenkinsfile.conda

```diff
@@ -97,10 +97,29 @@ pipeline {
                 }
             }
         }
+        stage('Trigger Job - Salobj Conda Package') {
+            when { expression { return env.buildSalObjConda.toBoolean() } }
+            steps {
+                script {
+                    def RESULT = sh (returnStdout: true, script: 
+                    """
+                    docker exec ${container_name} sh -c "
+                    cd ~/package_dir/ && 
+                    source ~/miniconda3/bin/activate > /dev/null && 
+                    conda install -q -y setuptools_scm > /dev/null && 
+                    python -c 'from setuptools_scm import get_version; print(get_version())'"
+                    """).trim()
+                     
+                    echo "Starting the SalObj_Conda_package/tickets/DM-26702 job; sal_version: ${SAL_Version}, xml_version: ${XML_Version} idl_version: ${RESULT}"
+                    build propagate: false, job: 'SalObj_Conda_package/tickets%2FDM-26702', parameters: [booleanParam(name: 'develop', value: true), booleanParam(name: 'buildCSCConda', value: true), string(name: 'idl_version',value: RESULT )], wait: false
```

Comment:
please update the branch to be 'develop' and not the ticket branch.  That should be the permanent state, until things change.